src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Nav from './components/Nav';
+import Books from './components/Books';
+import Categories from './components/Categories';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Nav />
```

Comment:
- [ ] These components should be present in your project, kindly implement them so your project meets the requirements
   - [ ] `BookList` component
      - Renders a list of books using the `Book` component
      - Renders a form by using the `CreateNewBook` component
   - [ ] `Book` component
      - Takes the book data as a prop and renders the title and the author
      - Has a button for removing books (the button wouldn't remove the books *yet*, this is just a placeholder for now)
  - [ ] `CreateNewBook` component
      - Renders a form with inputs for the title and the author of the book
      - Has a button to submit (the button wouldn't add the book *yet*, this is just a placeholder for now)
  - [ ] `Categories` component
      - should display only a button with the text "Check status"

Reply:
@Ismailco I appreciate your time to review my project. i had those components that you told me to add before I did not know where they were hidden.